.github/workflows/_reusable.rust.yml

```diff
@@ -37,6 +37,33 @@ jobs:
     container:
       image: xaynetci/yellow:v8
     timeout-minutes: 20
+    env:
+      XAYN_INGESTION__NET__BIND_TO: 127.0.1.1:3030
+      INGESTION_URI: http://127.0.1.1:3030
+      XAYN_PERSONALIZATION__NET__BIND_TO: 127.0.1.1:3031
+      PERSONALIZATION_URI: http://127.0.1.1:3031
+    services:
+      elasticsearch:
+        image: elasticsearch:8.5.3
+        env:
+          - discovery.type=single-node
+          - xpack.security.enabled=false
+        options: >-
+          --health-cmd curl -sf 'localhost:9200/_cluster/health'
```

Comment:
You might still need to check for it having properly started by e.g. checking ` curl -sf -X OPTIONS "http://localhost:9200/index_name" `. I had some cases where `_cluster/health` was already ready but  it wasn't ready to create an index yet.